app/src/main/java/org/simple/clinic/registration/register/RegistrationLoadingScreen.kt

```diff
@@ -30,6 +32,16 @@ class RegistrationLoadingScreen(
   @Inject
   lateinit var screenRouter: ScreenRouter
 
+  private val events by unsafeLazy {
+    Observable
+        .merge(
+            screenCreates(),
+            retryClicks()
+        )
+        .compose(ReportAnalyticsEvents())
```

Comment:
If the controller didn't have `ReportAnalyticsEvents`, should we add it to new events stream during migration? 🤔

Reply:
Yes, this was missed in the initial screen implementation.